cost-analyzer/templates/kubecost-metrics-deployment-template.yaml

```diff
@@ -0,0 +1,266 @@
+{{- if .Values.kubecostMetrics }}
+{{- if .Values.kubecostMetrics.exporter }}
+{{- if or .Values.kubecostMetrics.exporter.enabled .Values.agent }}
```

Comment:
Is there any reason you would want to deploy an agent (`.Values.agent != null`) with metrics exporter disabled? If not, I think it'd be nice to do something more like:
```
IF
  .Values.agent OR
  (.Values.kubecostMetrics AND
   .Values.kubecostMetrics.exporter AND
   .Values.kubecostMetrics.exporter.enabled)

Reply:
This is a dumb `.or` and not necessary. By deploying using the values-agent.yaml, you'll enable both. This existed before that was added. This should simply be 
```
{{- if .Values.kubecostMetrics.exporter.enabled }}
```